app/javascript/packs/data/socialMediaLinks.js

```diff
@@ -0,0 +1,7 @@
+export const FACEBOOK_LINK = "https://www.facebook.com/sharer/sharer.php?u=https%3A//feedadoc.com/"
+
+export const TWITTER_LINK = "https://twitter.com/intent/tweet?text=Check%20out%20%40FeedaDoc,%20a%20tool%20that%20matches%20hospital%20workers%20fighting%20%23COVID19%20with%20local%20volunteers%20who%20want%20to%20help.%20Please%20share%20with%20doctors,%20nurses,%20and%20all%20other%20hospital%20workers%20who%20would%20appreciate%20some%20extra%20support!%20https%3A//feedadoc.com/"
+
+export const LINKEDIN_LINK = "https://www.linkedin.com/shareArticle?mini=true&url=https%3A//feedadoc.com/&title=Feed%20a%20Doc%3A%20Care%20providers%20need%20our%20help%20to%20fight%20COVID-19&summary=Check%20out%20FeedaDoc,%20a%20tool%20that%20matches%20hospital%20workers%20fighting%20%23COVID19%20with%20local%20volunteers%20who%20want%20to%20help.%20Please%20share%20with%20doctors,%20nurses,%20and%20all%20other%20hospital%20workers%20who%20would%20appreciate%20some%20extra%20support!%20https%3A//feedadoc.com/&source="
+
+export const EMAIL_LINK = "mailto:?subject=Check%20out%20%22Feed%20a%20Doc%22&body=Hi,%20I%20thought%20you%20might%20be%20interested%20in%20this%20tool%20built%20by%20volunteers%20to%20support%20care%20providers%20fighting%20COVID-19.%20It%20lets%20healthcare%20workers%20post%20a%20request%20for%20support%20and%20matches%20them%20with%20volunteers%20(either%20friends%20or%20people%20in%20their%20community).%20%0A%20https%3A//feedadoc.com/"
```

Comment:
Doesn't this need an email address?

Reply:
I'm not sure. When the user clicks on the icon, their native email app will open and they can enter the email address of the person they want to share this with. If this is what we want, then this might be ok.

Reply:
Oh! That makes sense.